english/marshmallow.md

```diff
@@ -30,7 +30,7 @@ But it sounds like it might be coming out of speakers.
 
 When you look away, a marshmallow hand attempts to grab the lighter.
 
-But you are too quick for the marshmellow!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.